# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/win/RemoteAccessible.cpp
# Commit: 6591f14bbc71
# Full Hash: 6591f14bbc713f11b6316712d931ecc19a443a5c
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-05 03:52:32
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 06dda9072cb7 -r 6591f14bbc71 accessible/ipc/win/RemoteAccessible.cpp
--- a/accessible/ipc/win/RemoteAccessible.cpp	Mon Oct 04 19:28:49 2021 +0000
+++ b/accessible/ipc/win/RemoteAccessible.cpp	Mon Oct 04 19:46:54 2021 +0000
@@ -220,6 +220,10 @@
 }
 
 nsIntRect RemoteAccessible::Bounds() {
+  if (StaticPrefs::accessibility_cache_enabled_AtStartup()) {
+    return RemoteAccessibleBase<RemoteAccessible>::Bounds();
+  }
+
   nsIntRect rect;
 
   RefPtr<IAccessible> acc;